app/protocol/flows/v5/blockrelay/ibd.go

```diff
@@ -268,6 +277,42 @@ func (flow *handleIBDFlow) negotiateMissingSyncerChainSegment() (*externalapi.Do
 	return syncerHeaderSelectedTipHash, highestKnownSyncerChainHash, nil
 }
 
+func (flow *handleIBDFlow) isViolatingBoundedMergeDepth(highestKnownSyncerChainHash *externalapi.DomainHash, relayBlock *externalapi.DomainBlock) (bool, error) {
+	// Note: we don't wait until the hard fork is activated before applying these rules in the p2p leve.
+	// If such side chain is created before the hard fork, it will be downloaded if it'll be merged to
+	// the selected chain by another node.
+	virtualSelectedParent, err := flow.Domain().Consensus().GetVirtualSelectedParent()
```

Comment:
Isn't it better practice to extract `flow.Domain().Consensus()` to a local variable to use in all calls ?